zproject/default_settings.py

```diff
@@ -358,8 +358,8 @@
 # Version number for ToS.  Change this if you want to force every
 # user to click through to re-accept terms of service before using
 # Zulip again on the web.
-TOS_VERSION: Optional[str] = None
-# Template to use when bumping TOS_VERSION to explain situation.
+TERMS_OF_SERVICE_VERSION: Optional[str] = None
+# Template to use when bumping TERMS_OF_SERVICE_VERSION to explain situation.
 FIRST_TIME_TOS_TEMPLATE: Optional[str] = None
```

Comment:
We could also expand the TOS in here, if we're doing it for `TOS_VERSION`

Reply:
Good idea.  Added a comment to do this, and also remove the implied `zerver/` prefix on the template path.